src/software/logger/network_sink.cpp

```diff
@@ -21,12 +22,22 @@ NetworkSink::NetworkSink(unsigned int channel, const std::string& interface, int
 
 void NetworkSink::sendToNetwork(g3::LogMessageMover log_entry)
 {
-    auto level = log_entry.get()._level;
+    g3::LogMessage new_log = log_entry.get();
+    for (g3::LogMessage log : log_merger.log(new_log))
+    {
+        sendOneLogToNetwork(log);
+    }
```

Comment:
The compiler might fix this, but we should try to reduce the number of string copies
```suggestion
    for (const g3::LogMessage &log : log_merger.log(new_log))
    {
        sendOneLogToNetwork(log);
    }
```